### PR TITLE
ft(CLDSRV-102): Add Aborted MPU PUT

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

File: lib/metadata/bucketfile/backend.js [hunk 1]
```diff
@@ -190,6 +190,9 @@ class BucketFileInterface {
             if (err) {
                 return cb(err);
             }
+            if (params && params.isAbort && params.replayId) {
```
Comment (jonathan-gramain) [inline lib/metadata/bucketfile/backend.js]: Maybe comment here that we don't handle abort specifically but just ignore it.

Also any abort could be ignored, there should not be a need to check `params.replayId` (even though we normally always pass that attribute, it's an implementation detail).

**Review by jonathan-gramain — APPROVED** [review]

**Review by jonathan-gramain — APPROVED** [review]

**Review by naren-rajendran — APPROVED** [review]

**Pull request merged**